server/src/routes/api/v1/usersRouter.js

```diff
@@ -22,7 +25,41 @@ usersRouter.post("/", async (req, res) => {
     throw error;
   }
 
+  // const fuzziesAsElevator = await newUser.$relatedQuery("givenWarmFuzzies")
+
   return res.status(201).json({ user: newUser });
 });
 
+usersRouter.get("/:id/warm-fuzzies/given", async (req, res) => {
+  const user = await User.query().findById(req.params.id);
+  const elevations = await WarmFuzzy.query().select('createdAt', 'title', 'description', 'recipientId', 'elevatorId').where('elevatorId', `${user.id}`);
```

Comment:
I would probably avoid the explicit select here. Just let all the properties flow through.